test/e2e2/spec/test_api_rr.py

```diff
@@ -40,7 +40,7 @@ def generate_params(dict_value_lists_by_type: dict) -> List[Tuple[str, str]]:
         '6454 8 1 24396e17e36d031f71c354b06a979a67a01f503e',
         '0 0 0 00',
     ],
-    'CERT': ['6 0 0 sadfdQ=='],
+    'CERT': [],  # handled in non-canonical, because apply presentation format conversion on the stack (but not here)
```

Comment:
Add a canonical example. (I do not understand if/why there can't be one.)

Reply:
It's not possible, because we have an override for the presentation format conversion of the CERT rrtype in dns.py on the stack (see above), but not in the testing container. We could only do this if we also mock dnspython in the test env. Not sure if that makes sense.

The non-canonical tests cover this type too, so I feel nothing is really missing.

Reply:
alright :+1: